app/models/project.js

```diff
@@ -20,19 +22,46 @@ export const EmptyFeatureCollection = {
   }],
 };
 
+// (property,key) basic checks
 const hasAnswered = property => property === true || property === false;
 const hasFilledOut = property => !isEmpty(property);
+const hasFilledOutAndProposedDifferingZoning = function(property, key) {
+  debug(`
+    check;
+      ${key}
+      is it empty? ${!isEmpty(property)}
+  `);
+  return !isEmpty(property) && this.get(`${key}Model.proposedDiffersFromCanonical`);
```

Comment:
maybe comment what `${key}Model` ends up referencing? or at the top of the function comment about what `key` and `property` can be expected to be

Reply:
Will do. This is a little hacky and obviously brittle.